MyCryptoHQ/MyCrypto
# Remove invalid prop warning

[ch6443]

<img width="612" alt="Screenshot 2020-07-20 at 17 46 47" src="https://user-images.githubusercontent.com/2429708/87968637-059e9800-cac1-11ea-82b6-27704ac52fb4.png">


Steps to test:
1. Head to `/dashboard` on beta.mycrypto.com
2. Open the console and see the error.
3. Repeat on mycryptobuilds.com of the PR and see that it is no longer present.
